taurus/entity/value/tests/test_uniform_real.py

```diff
@@ -0,0 +1,24 @@
+"""Tests of the UniformReal class."""
+import pytest
+
+from taurus.entity.value.uniform_real import UniformReal
+
+
+def test_bounds_order():
+    """Lower bound must be <= upper bound."""
+    UniformReal(4.4, 8.8, 'm')
+    UniformReal(100.0, 100.0, 'm')
+    with pytest.raises(AssertionError):
+        UniformReal(23.2, 18.9, 'm')
+
+
+def test_equality():
+    """Test that equality checks both bounds and integers."""
+    value1 = UniformReal(0, 1, '')
+    value2 = UniformReal(0, 100, 'cm')
```

Comment:
Do we want to check pint functionality?

Reply:
Fair. I updated it to be a simple check of the units strings (which __eq__ does compare).